actions/fight_preparations.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+from characters.knights import Knights
+from items.potion import Potion
+from items.armour import Armour
+from items.weapon import Weapon
+
+
+class FightPreparations:
+
+    def __init__(self) -> None:
+        pass
+
+    @staticmethod
+    def apply_potion(knight: Knights) -> None:
```

Comment:
provide potion into method as parameter, instead of doing loop
Fix with all functions